completions/bash/ocitools

```diff
@@ -209,6 +209,8 @@ _ocitools_ocitools() {
 	local boolean_options="
 		$global_boolean_options
 		--help
+		--host-specific
+		--log-level
```

Comment:
This is [not a boolean option][1] (although we probably want to update the man page to have `**--log-level**=LEVEL` to reflect that.

[1]: https://github.com/opencontainers/ocitools/blob/6aeb752f0c9c4142a50b00d872b2b6d0c400dd0f/cmd/ocitools/main.go#L16-L20